security/libu2f-host/Portfile

```diff
@@ -27,7 +27,7 @@ checksums           rmd160  eec9e62e6a03348bf5d651bc4a29b112dd9b35be \
 
 depends_build       port:pkgconfig
 depends_lib         port:json-c \
-                    port:hidapi-devel
+                    port:hidapi
```

Comment:
Revbump needed.

upgrade:
```
--->  Installing hidapi @0.8.0_0
--->  Cleaning hidapi
--->  Unable to deactivate hidapi-devel @0.8.0-20160920_0, the following ports depend on it:
--->  	libu2f-host @1.1.7_0
Warning: Deactivate forced.  Proceeding despite dependencies.
--->  Deactivating hidapi-devel @0.8.0-20160920_0
--->  Cleaning hidapi-devel
--->  Activating hidapi @0.8.0_0
--->  Cleaning hidapi
```
reclaim:
```
--->  Checking for unnecessary unrequested ports
Unrequested ports without requested dependents found: hidapi  @0.8.0_0
Would you like to uninstall them? [Y/n]:
--->  Deactivating hidapi @0.8.0_0
--->  Cleaning hidapi
--->  Uninstalling hidapi @0.8.0_0
--->  Checking for inactive ports
Inactive ports found: hidapi-devel  @0.8.0-20160920_0
Would you like to uninstall them? [Y/n]:
--->  Unable to uninstall hidapi-devel @0.8.0-20160920_0, the following ports depend on it:
--->  	libu2f-host @1.1.7_0
Warning: Uninstall forced.  Proceeding despite dependencies.
--->  Uninstalling hidapi-devel @0.8.0-20160920_0
--->  Building list of distfiles still in use
--->  Searching for unused distfiles
No unused files found.
--->  Scanning binaries for linking errors
--->  Found 2 broken files, matching files to ports
--->  Found 1 broken port, determining rebuild order
You can always run 'port rev-upgrade' again to fix errors.
The following ports will be rebuilt: libu2f-host @1.1.7
```

Reply:
Sorry, I wasn't sure whether incrementing the revision was necessary in this case, and neglected to ask. It did not occur to me that already-installed ports "remember" their dependencies even if their portfiles get updated without the version/revision incrementing.

I have opened another PR: #4208